src/Symfony/Component/Workflow/Event/GuardEvent.php

```diff
@@ -34,12 +34,12 @@ public function __construct($subject, Marking $marking, Transition $transition,
         $this->transitionBlockerList = new TransitionBlockerList();
     }
 
-    public function isBlocked(): bool
+    public function isBlocked()
     {
         return !$this->transitionBlockerList->isEmpty();
     }
 
-    public function setBlocked(bool $blocked): void
+    public function setBlocked(bool $blocked)
```

Comment:
the argument type hint needs to be removed too

Reply:
:man_facepalming: Good catch. Sorry about this waste of time.